src/main/java/frc/robot/Constants.java

```diff
@@ -59,4 +59,21 @@ public final class Constants {
     public static final double FRONT_LEFT_ABS_ENCODER_ZERO = 339.5;
     public static final double BACK_LEFT_ABS_ENCODER_ZERO = 18.72;
     public static final double FRONT_RIGHT_ABS_ENCODER_ZERO = 133.3;
+
+    public static final double ArmSeperationDistance = 5.00; 
```

Comment:
Why does your PR have INTAKE constants? Please remove any constant not related to the code in this PR.